src/tracing/trace_event.cc

```diff
@@ -3,14 +3,14 @@
 namespace node {
 namespace tracing {
 
-v8::Platform* platform_ = nullptr;
+v8::TracingController* controller_ = nullptr;
```

Comment:
Since you're here: `g_controller`?  We use underscore suffix convention really only for data members.